tests/source/control-brace-style-always-next-line.rs

```diff
@@ -2,33 +2,33 @@
 
 fn main() {
     loop {
-        ();
-        ();
+        let foo = ();
```

Comment:
@calebcartwright Why do we need to update all the tests?

Reply:
Sorry i meant to leave a note on that. Apparently both `;` and `();` are parsed as empty tuple exprs in the block statements, so with the updated logic filtering out empty statements, these blocks were now being detected as empty.  

```rust
    loop {
        ();
        ();
        }
```

would be flattened to 
```rust
    loop {}
```

which was making these tests fail since they're trying to validate brace placement.

Is this the correct behavior? Should we treat a block that only has a `();` statement the same as one that only has a `;`?

Reply:
Thank you for your explanation!

> Is this the correct behavior? Should we treat a block that only has a (); statement the same as one that only has a ; (both as empty blocks)?

Removing `();` seems fine to me, though I am afraid that we may need to version-gate this as we didn't use to remove it. As we will be preparing for 2.0 release in the near future, I would like to merge this PR after we released 1.4.10 so that we don't need to add an extra version-gate which will be removed soon.

Reply:
SGTM!